src/signals/incident-management/containers/IncidentDetail/types.ts

```diff
@@ -65,7 +65,7 @@ export interface HistoryEntry {
 export interface Attachment {
   _display: string
   created_at: string
-  created_by: string
+  created_by: string | null
```

Comment:
why not optional?

Reply:
This is more precise. The back-end returns a null if there is no created_by.